GeosCore/fullchem_mod.F90

```diff
@@ -3040,6 +4242,316 @@ SUBROUTINE Init_FullChem( Input_Opt, State_Chm, State_Diag, RC )
        RETURN
     ENDIF
 
+#ifdef MODEL_GCHP
```

Comment:
I would prefer this implementation to be a compile-time option so that we can turn it off for testing.